src/java.base/share/classes/sun/security/provider/SecureRandom.java

```diff
@@ -65,10 +65,14 @@
     private int remCount;
 
     /**
-     * This empty constructor automatically seeds the generator.  We attempt
-     * to provide sufficient seed bytes to completely randomize the internal
-     * state of the generator (20 bytes).  Note, however, that our seed
-     * generation algorithm has not been thoroughly studied or widely deployed.
+     * This empty constructor.
```

Comment:
How about "An empty constructor that creates an unseeded SecureRandom object"?

Reply:
Sure, that sounds great.